### DISCUSSION TITLE
Locale not working in transaction creation

### DISCUSSION THREAD

**Discussion by bannisdale** [firefly-iii/firefly-iii#4499] [Firefly III]
**Bug description**
I am running Firefly III version 5.4.6, and my problem is: I have chosen the language English but locale is German (Germany). Therefore, dates and amounts are correctly displayed as

"12.03.2021 - 99,99 €".

However, when I create or edit a transaction, it looks like this:

![image](https://user-images.githubusercontent.com/42511384/110950723-79d5c680-8344-11eb-81f7-e2f02d25214c.png)

**Steps to reproduce**

- create a transaction
- try to enter an amount based on the German locale (with , as delimiter) OR look at the date order

**Extra info**
This is by far not a big issue, but an inconsistency that sometimes bugs me a bit. :D

**Comment by JC5** [comment]
Is your browser or OS set to English, by any chance?

**Reply by bannisdale** [reply]
Thanks for the quick response!

Yeah, well, given the error message and the overall display of the locale, I somehow didn't consider that the form fields might be influenced by the browser. But yes, if I change my browser to German, both fields work with the locale as expected.

I'm not sure if this is working as intended or could even be changed (override browser setting with the locale set in the settings). Honestly, I'd like it to be, but I totally understand that this is mostly a problem I created for myself. :D
